20241002.md

```diff
@@ -0,0 +1,47 @@
+# RISE RP005 QEMU weekly report 2024-10-02
+
+## Work completed since last report
+
+[SiFive function performance report](https://drive.google.com/file/d/1kr5vCa75Bdy9MtF781KLEplOXlOxbwwL).
+
+- WP2
+  - Complete SiFive library function set
+    - currently in progress.
+  - Single instruction benchmarking relevant to SiFive library functions
+    - on hold.
+  - Resume work on AArch64 comparison.
+    - working on the single instruction comparison between a Aarch64 vector load and vle8.v.
+    - on hold to prioritize the separation of the patches.
+
+- Other
+  - Separate Embecosm's and Max's patches:
+    - The first patch (loop optimization in `vext_ldst_us`) is extracted.
+    - The second patch (large loads and stores optimization through `__builtin_memcpy`) requires work.
```

Comment:
Suggest coallesc this into just
- work in progress.  This is a substantial effort because all the changes since July have been assumed to build on Max's work, and hence we need to write code to allow the patch to work standalone.